docs/tables/aws_cloudwatch_log_group.md

```diff
@@ -45,3 +43,13 @@ from
   aws_cloudwatch_log_group groups
   join aws_cloudwatch_log_metric_filter metric on groups.name = metric.log_group_name;
 ```
+
+### List the cloudwatch log groups' data protection policy documents
```

Comment:
This example seems a bit lackluster, can you please add more examples, similar to what we have in S3 bucket (https://hub.steampipe.io/plugins/turbot/aws/tables/aws_s3_bucket#list-buckets-that-enforce-encryption-in-transit? Since the policy format is a bit different than other resource policies, I think there's an opportunity to create some good starting examples here.